ocaml/xapi-cli-server/record_util.ml

```diff
@@ -150,6 +150,38 @@ let string_to_vm_operation x =
   else
     List.assoc x table
 
+let vm_uefi_mode_of_string = function
+  | "setup" ->
+      `setup
+  | "user" ->
+      `user
+  | s ->
+      raise (Record_failure ("Expected 'user','setup', got " ^ s))
```

Comment:
"Expected one of 'user', 'record' but got '%s'"

Reply:
I'm replicating how other Record_failure are written, I think it's better to keep the message standarize.